packages/common/src/tokens/token.ts

```diff
@@ -16,8 +17,9 @@ export class Token {
   readonly decimals: number;
   readonly symbol: string;
   readonly name: string;
+  readonly logoURI: string | undefined;
```

Comment:
```suggestion
  readonly logoUri?: string;
```

Simplify it and currently we use camelCase for naming, such as `chainId`, `protocolId` and `lstApy`.

Reply:
fixed